envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/EnvoySnapshotFactory.kt

```diff
@@ -213,12 +214,13 @@ class EnvoySnapshotFactory(
     }
 
     private fun getServicesEndpointsForGroup(
+        rateLimitEndpoints: List<IncomingRateLimitEndpoint>,
         globalSnapshot: GlobalSnapshot,
         egressRouteSpecifications: Collection<RouteSpecification>
-    ): List<ClusterLoadAssignment> {
-        return egressRouteSpecifications
-            .mapNotNull { globalSnapshot.endpoints.resources()[it.clusterName] }
-    }
+    ): List<ClusterLoadAssignment> =
+        (egressRouteSpecifications.map { it.clusterName } +
```

Comment:
this is not readable for me. Too much is happening in this two lines. Can you split it and make it easier to digest?

Reply:
Fixed.